launch/launch/actions/include_launch_description.py

```diff
@@ -69,6 +74,26 @@ def __init__(
         self.__launch_description_source = launch_description_source
         self.__launch_arguments = launch_arguments
 
+    @staticmethod
+    def parse(entity: Entity, parser: Parser):
+        """Return `IncludeLaunchDescription` action and kwargs for constructing it."""
+        _, kwargs = super(
+            IncludeLaunchDescription,
+            IncludeLaunchDescription
+        ).parse(entity, parser)
+        file = parser.parse_substitution(entity.get_attr('file'))
```

Comment:
@ivanpauno nit: I know I brought this up before, and that `file` is no longer a Python keyword, but I can count 3 syntax highlighters that think otherwise. Can we rename it?